scalafix-core/shared/src/main/scala/scalafix/internal/config/CustomMessage.scala

```diff
@@ -0,0 +1,29 @@
+package scalafix.internal.config
+
+import metaconfig.{Conf, ConfError, ConfDecoder, Configured}
+import org.langmeta._
+import MetaconfigPendingUpstream.XtensionConfScalafix
+
+import scala.language.implicitConversions
+
+case class CustomMessage[T](value: T, message: Option[String])
+object CustomMessage {
+  def decoder[T](field: String)(
+      implicit ev: ConfDecoder[T]): ConfDecoder[CustomMessage[T]] =
+    ConfDecoder.instance[CustomMessage[T]] {
+      case obj: Conf.Obj =>
+        (obj.get[T](field) |@| obj.get[String]("message")).map {
+          case (value, message0) => {
+            val message =
+              if (message0.split(System.lineSeparator).size > 1) {
+                System.lineSeparator + message0.stripMargin
```

Comment:
Is stripMargin necesary? We I think a `trim` should be enough, so you can write configuration like this
```
Disable {
  message = """
Are you sure?

blah
blah
"""
}
```

Reply:
both would work with stripMargin

Reply:
Aha, of course 😅  sure we can keep stripMargin, but I think we should advertise the non-stripMargin syntax in the docs since it's less annoying IMO.